Sources/TuistKit/Commands/Project/ProjectShowCommand.swift

```diff
@@ -0,0 +1,43 @@
+import ArgumentParser
+import Foundation
+import Path
+import TuistSupport
+
+struct ProjectShowCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "show",
+            _superCommandName: "project",
+            abstract: "Show information about the specified project. Use --web flag to open the project in the browser."
+        )
+    }
+
+    @Argument(
+        help: "The project to show. The full handle must be in the format of account-handle/project-handle.",
+        completion: .directory,
+        envKey: .projectShowFullHandle
+    )
+    var fullHandle: String?
+
+    @Option(
+        help: "Open a project in the browser.",
+        envKey: .projectShowWeb
+    )
+    var web: Bool = false
```

Comment:
I learned recently through @natanrolnik's blog post that you can configure flags to support [inversion](https://swifttoolkit.dev/posts/argument-parser-guide-2#flag-inversion). What do you think if we use that feature here?

Reply:
I don't think it makes sense here. It would for something like `tuist generate --no-open` since the default action is yes, but here, the inversion is already the default.